decentraland/unity-explorer
# Fix: camera reel capture view shortcuts panel is missing key icons

## What does this PR change?
Somehow reference for outline icon was lost on the prefab, so I re-added it with slight modifications:

- Re-organized prefab structure for Shortcuts panel
- Re-added outline for shortcut keys 
- Added shortcuts for tilting and right mouse click

## How to test the changes?

1. Launch the explorer with debug and camera reel CLI args
2. Open Screencapture view and open shortcuts panel there
3. Verify that all b

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
